plugin/src/main/java/com/h3xstream/findsecbugs/injection/BasicInjectionDetector.java

```diff
@@ -79,6 +89,63 @@ public void receiveInjectionPoint(String fullMethodName, InjectionPoint injectio
     }
 
     /**
+     * Loads taint sinks from custom file. The file name is passed using system property based on the current class name.<br />
+     * <br />
+     * Example for Linux/Mac OS X:<ul>
+     *     <li>-Dfindsecbugs.injection.customconfigfile.SqlInjectionDetector="/tmp/sql-custom.txt|SQL_INJECTION_HIBERNATE:/tmp/sql2-custom.txt|SQL_INJECTION_HIBERNATE"</li>
+     *     <li>-Dfindsecbugs.injection.customconfigfile.ScriptInjectionDetector="/tmp/script-engine-custom.txt|SCRIPT_ENGINE_INJECTION:/tmp/el-custom.txt|EL_INJECTION"</li>
+     * </ul>
+     * Example for Windows:<ul>
+     *     <li>-Dfindsecbugs.injection.customconfigfile.SqlInjectionDetector="C:\Temp\sql-custom.txt|SQL_INJECTION_HIBERNATE;C:\Temp\sql2-custom.txt|SQL_INJECTION_HIBERNATE"</li>
+     *     <li>-Dfindsecbugs.injection.customconfigfile.ScriptInjectionDetector="C:\Temp\script-engine-custom.txt|SCRIPT_ENGINE_INJECTION;C:\Temp\el-custom.txt|EL_INJECTION"</li>
+     * </ul>
+     */
+    protected void loadCustomConfigFiles() {
+        String customConfigFile = FindSecBugsGlobalConfig.loadFromSystem("findsecbugs.injection.customconfigfile." + getClass().getSimpleName(), null);
```

Comment:
I would prefer that the specialized configuration would be obtain from a non-static method. `loadFromSystem()` could be private.
The singleton pattern can be quite useful when testing the effect of configuration changes.

For now:
```java
FindSecBugsGlobalConfig.getInstance().getCustomConfigFile( getClass().getSimpleName() )
```
If the configuration gets tested:
```java
protected FindSecBugsGlobalConfig config = FindSecBugsGlobalConfig.getInstance()
[...]
config.getCustomConfigFile( getClass().getSimpleName() );
```